lib/src/utils/types.ts

```diff
@@ -68,6 +68,7 @@ export type TChannelManagerClaim = {
 };
 
 export type TChannelManagerPaymentSent = {
+	bolt11_invoice: string;
```

Comment:
Make it an optional rather so Bitkit knows it's not always there. As any payments made prior to this update won't have that set.

Reply:
Cool! Done, and thank you!